config/routes.rb

```diff
@@ -28,6 +28,7 @@
     end
 
     namespace :admin do
+      resources :data_imports, except: [:index]
```

Comment:
We have this nested under `source_files` -- does it need to be here too?

Reply:
@littleforest yes. Removing this will throw an error even thought the path is not in used. See similar issue [here](https://github.com/thoughtbot/administrate/issues/899#issuecomment-787523764)